zensu fails if no dependency is installed

fhdk opened the pull request:
@Chrysostomus  @yochananmarqos 
E.g. zenity is the last dependency checked but if it is not installed then the result is

```
$ zensu pcmanfm
/usr/bin/zensu: line 10: zenity: command not found
triggered empty
```

fhdk committed:
added dependency fail

fhdk committed:
fix spelling

yochananmarqos commented:
@fhdk There's a typo on line 9:

```
-elif [ -e /usr/bin/zenity ] then;
+elif [ -e /usr/bin/zenity ]; then
```

fhdk committed:
Update zensu.sh

fix typo semicolon the wrong place in `elif` clause
removed unnecessary `=` from zneitia dialog title
